src/style.css

```diff
@@ -1 +1,83 @@
 /* add styles here */
+@font-face {
+  font-family: 'Avenir normal';
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Avenir bold';
+  src: url(/src/fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+  font-style: bold;
```

Comment:
The `font-weight` for 'Avenir bold' is set to 300, which is typically a light weight. Consider using a heavier weight like 700 or 900 for bold fonts. Also, `font-style` should be `normal` instead of `bold` as `font-style` is used for italic or normal styles, not weight.